reg.py

```diff
@@ -8,6 +8,7 @@ def hello_world():
 def hello():
     if request.method == 'POST':
       data = request.form
+      print("Email Id:"+data["email"])
```

Comment:
are you using this file, if not, delete them.

Reply:
just kept if it is required afterwards